.github/workflows/installer.yml

```diff
@@ -0,0 +1,53 @@
+name: Installer
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master", "qt6" ]
+
+jobs:
+  windows:
+    name: Qt 6.5.0 / Windows
+    runs-on: windows-2022
+    steps:
+      - name: Clone the repository
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: Setup CMake
+        uses: ./.github/actions/cmake
+        with:
+          qt_arch: win64_mingw
+          additional_cmake_args: -DCMAKE_INSTALL_PREFIX='${{ github.workspace }}/install'
+
+      - name: Build
+        run: cmake --build '${{github.workspace}}/build' --parallel "$(nproc)"
+
+      - name: Install
+        run: cmake --install '${{ github.workspace }}/build'
+
+      - name: Run windeployqt
+        # --no-translations is needed because https://bugreports.qt.io/browse/QTBUG-112204
+        # FIXME: Remove when 6.5.1 becomes available.
+        run: windeployqt --no-translations -serialport '${{ github.workspace }}/install/bin/quickevent.exe'
+
```

Comment:
I think `libpq` dll is missing. Make sure it is possible to connect to Postgres database.

Reply:
I'll merge this, even if not finished to push things forward, libpq and others will be solved in new issues.